core/shared/platform/windows/platform_internal.h

```diff
@@ -137,6 +137,12 @@ bh_atomic_thread_fence(int mem_order);
 
 #define os_atomic_thread_fence bh_atomic_thread_fence
 
+#define WASI_LIBC_INVALID_RAW_HANDLE INVALID_HANDLE_VALUE
+
+typedef HANDLE os_handle;
+typedef void os_dir_stream;
+typedef HANDLE os_raw_handle;
```

Comment:
`os_raw_handle` is the same type `os_handle` but that won't be true when this is actually implemented for Windows because we'll need to store some other info in `os_handle` which we can't get easily from the native `HANDLE`.